experimental/addons/dapr/readme.md

```diff
@@ -0,0 +1,3 @@
+# dapr
+
+This addon provides dapr support for pubsub.
```

Comment:
```suggestion
This addon provides dapr support for pub/sub.
```

Can you proviode more introduction about this addon? let's add some examples about how to use and add some links.